src/scripts/achievements/AchievementHandler.ts

```diff
@@ -41,14 +41,19 @@ class AchievementHandler {
         return this.achievementListFiltered().slice(this.navigateIndex() * 10, (this.navigateIndex() * 10) + 10);
     }
 
-    public static filterAchievementList() {
+    public static filterAchievementList(retainPage?: boolean) {
         this.achievementListFiltered(this.achievementList.filter((a) => (
             a.region <= player.highestRegion() &&
-            (this.filter.status() == 'all' || a.unlocked == JSON.parse(this.filter.status())) &&
-            (this.filter.type()   == 'all' || a.property.constructor.name == this.filter.type()) &&
-            (this.filter.region() == 'all' || a.region == +this.filter.region())
+              (this.filter.status() == 'all' ? true : a.unlocked == JSON.parse(this.filter.status())) &&
+              (this.filter.type()   == 'all' ? true : a.property.constructor.name == this.filter.type()) &&
+              (this.filter.region() == 'all' ? true : a.region == +this.filter.region())
```

Comment:
Is there any reason for this change?
As the logic still seems to be the same.

Reply:
maybe some of merges did that, i dont edited this part

Reply:
solved